common/djangoapps/util/log_sensitive.py

```diff
@@ -1,208 +0,0 @@
-"""
-Utilities for logging sensitive debug information such as authentication tokens.
-
-Usage:
-
-1. Generate keys using ``python3 -m common.djangoapps.util.log_sensitive gen-keys``
-2. Follow the instructions it prints out, and pay close attention to the warning
-   at the end of the output
-3. When logging sensitive information, use like so::
-
-     logger.info(
-         "Received invalid auth token %s in Authorization header",
-         encrypt_for_log(token, getattr(settings, 'YOUR_DEBUG_PUBLIC_KEY', None))
-     )
-
-   This will log a message like::
-
-     Received invalid auth token [encrypted: ZXI...fFo=|IYS...1KA==] in Authorization header
-
-4. If you need to decrypt one of these messages, save the encrypted portion
-   to file, retrieve the securely held private key, and run
-   ``python3 -m common.djangoapps.util.log_sensitive decrypt --help``
-   for instructions.
-"""
-
-from base64 import b64decode, b64encode
-
-import click
-from nacl.public import Box, PrivateKey, PublicKey
```

Comment:
You introduce PyNaCl in https://github.com/openedx/edx-platform/pull/29682/files. Do you want to remove that dependency?